From d3204a25f9ff1ae04c60507108bce2988db34c05 Mon Sep 17 00:00:00 2001
From: Octavian Purdila <octavian.purdila@nxp.com>
Date: Fri, 3 Mar 2017 13:04:22 +0200
Subject: [PATCH 1547/5242] MLK-14304 ARM: dts: imx7d: enable ocotp node

commit  01d611484b0242f740d82f168f701ff665c3d253 from
https://source.codeaurora.org/external/imx/linux-imx.git

The octop node was defined in imx7s.dtsi but disabled. It is needed by
the fsl_otp driver.

Signed-off-by: Octavian Purdila <octavian.purdila@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7d.dtsi |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/imx7d.dtsi b/arch/arm/boot/dts/imx7d.dtsi
index 61ed893..de4f47e 100644
--- a/arch/arm/boot/dts/imx7d.dtsi
+++ b/arch/arm/boot/dts/imx7d.dtsi
@@ -172,6 +172,7 @@
 		compatible = "fsl,imx7d-ocotp", "syscon";
 		reg = <0x30350000 0x10000>;
 		clocks = <&clks IMX7D_OCOTP_CLK>;
+		status = "okay";
 	};
 
 	tempmon: tempmon {
-- 
1.7.9.5

